Src/Managers/size_manager.py

```
@@ -0,0 +1,102 @@
+            return height, width
+
+        current_font_size: int = self.__font_manager.get(item).size
+        min_font_size = get_userdata(item, 'min_font_size').size
+        font_ratio = min_font_size / current_font_size
+        height = set_userdata(item, 'min_height', (dpg.get_item_height(item) or 0) * font_ratio)
+        width = set_userdata(item, 'min_width', (dpg.get_item_width(item) or 0) * font_ratio)
+        return height, width
+
+
+    def set_bbox(self, item: int | str, height: int, width: int):
+        if DPGType(dpg.get_item_type(item)) is DPGType.TEXT: return # У текста бл*ть есть ширина, которую нельзя изменять, великолепно нахуй
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)